whois-api/src/test/java/net/ripe/db/whois/api/autocomplete/AutocompleteServiceTestIntegration.java

```diff
@@ -33,7 +34,7 @@
 import static org.hamcrest.Matchers.is;
 import static org.junit.jupiter.api.Assertions.fail;
 
-@org.junit.jupiter.api.Tag("IntegrationTest")
+@Tag("IntegrationTest")
```

Comment:
This change would be better in a separate branch. Could you create a separate PR for that? It would reduce the changes in this PR a lot (and you could merge that one first).

Reply:
I think this refactoring is related to this PR. I committed in one go with lot of other changes so I need to revert a lot and earlier I had to use fully qualified name as there was another Tag class, now its deleted with this PR. Its just the number of classes else its a one liner change

Reply:
But if you still insist on creating another PR let me know I can spend some more time to separate them

Reply:
Thanks for explaining @maggarwal13 I agree with you, keep everything in one PR the changes are related.